### PR TITLE
feat(config): Add forceOverwrite option to reference.yml

### PR DESCRIPTION
**Pull request by dineshr93**
Signed-off-by: Dinesh Ravi <dineshr93@gmail.com>

follow up of https://github.com/oss-review-toolkit/ort/pull/8003

1. removed merge commit
2. Inserted `forceOverwrite` in alphabhetical order as @mnonnenmacher  mentioned
3. Added `ortConfig.forceOverwrite shouldBe true` in OrtConfigurationTest.kt

Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

### CODE REVIEW

File: model/src/main/resources/reference.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ ort:
   enableRepositoryPackageConfigurations: true
   enableRepositoryPackageCurations: true
 
+  # Force overwriting of any existing output files
```
Comment (sschuberth) [inline model/src/main/resources/reference.yml:34]: Please add a dot at the end of this sentence. Do this by amending the commit, but by adding a new commit that just adds the dot.

**Review by sschuberth — CHANGES_REQUESTED** [review]
The formatting of your `Signed-off-by` footer line is wrong. Instead of

```
feat(config): Add forceOverwrite option to reference.yml
 Signed-off-by: Dinesh Ravi <dineshr93@gmail.com>
```

it needs to be

```
feat(config): Add forceOverwrite option to reference.yml

Signed-off-by: Dinesh Ravi <dineshr93@gmail.com>
```

**Comment by dineshr93** [comment]
@sschuberth  Done

File: model/src/main/resources/reference.yml (hunk 1 above)
Comment (sschuberth) [reply model/src/main/resources/reference.yml]: > by adding a new commit that just adds the dot.

@dineshr93 you didn't amend you existing commit, but you added another commit. That is, there are not two commit on one in this PR. We'd like review comments to be addressed in the commit that the review comment belongs to.

**Comment by dineshr93** [comment]
@sschuberth  I have rebased to latest main, made into single commit and did proper signed off  with `git commit -s` after configuring gpg keys https://stackoverflow.com/a/67971184/2018343 . Before I was manually signing off in commit message

File: model/src/main/resources/reference.yml (hunk 1 above)
Comment (dineshr93) [reply model/src/main/resources/reference.yml]: > by adding a new commit that just adds the dot.

@sschuberth I thought you were asking me to add a new commit that just adds the dot. but any way I have now made into single commit.

Comment (sschuberth) [reply model/src/main/resources/reference.yml]: Sorry, my original post was missing the "not". I've added it for reference.

**Comment by sschuberth** [comment]
> did proper signed off with `git commit -s` after configuring gpg keys https://stackoverflow.com/a/67971184/2018343 .

You're mixing adding a `Signed-off-by` footer (`git commit -s`; this is what we require) with crytographically signing a commit (`git commit -S`, note the upper-case "S"; we don't require that).

**Comment by dineshr93** [comment]
Hi @sschuberth  I am not sure why one automated check failed `Build and Test/funTest-docker`

**Comment by sschuberth** [comment]
> I am not sure why one automated check failed `Build and Test/funTest-docker`

Those are test failures not related to your changes.

**Comment by sschuberth** [comment]
Sorry for not noticing this earlier:

> feat(config): Add forceOverwrite option to reference.yml

The conventional commit message type should be rather "docs" than "feat". Please amend the commit message accordingly.

**Comment by dineshr93** [comment]
> The conventional commit message type should be rather "docs" than "feat". Please amend the commit message accordingly.
amended the commit

**Review by sschuberth — APPROVED** [review]

**Pull request merged**